rq/defaults.py

```diff
@@ -3,4 +3,5 @@
 DEFAULT_WORKER_CLASS = 'rq.Worker'
 DEFAULT_CONNECTION_CLASS = 'redis.StrictRedis'
 DEFAULT_WORKER_TTL = 420
+DEFAULT_WORKER_BUSY_TTL = 45
```

Comment:
Can we call this `DEFAULT_JOB_MONITORING_INTERVAL` and change it to `30`? Workers can then call `heartbeat(DEFAULT_WORKER_BUSY_TTL + 15)`.

This is easier to explain and document if we choose to make the monitoring interval configurable in the future.